src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_VALUE = 100;

```

Comment:
The constant 'MAX_VALUE' is well named according to the style guide, which is good. However, ensure that all magic numbers in your code should be constants. If there are any other magic numbers in the project, they should be replaced with named constants as well.